### DISCUSSION TITLE
How to calculate the asymptomatic cases in China

### DISCUSSION THREAD

**Discussion by ZauberViolino** [owid/covid-19-data#2583] [Q&A] [answered]
I don't think the government of PR China is publishing total asymptomatic cases nor if any of those cases had turned symptomatic. So how did you count them?

**Comment by edomt** [comment, marked as answer]
See https://github.com/CSSEGISandData/COVID-19/issues/5579

**Reply by ZauberViolino** [reply]
Ah thanks!
